starwars/src/components/DisplayPeople.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StarWarsCard = styled.div`
+    width: 30%;
+    background: black;
+    border: 2px solid white;
+    border-radius: 10px;
+    margin: 20px;
+    padding: 5px;
+`
+
+const Name = styled.h2`
+    font-size: 30px;
+    text-align: center;
+    color: white;
+`
+
+const Info = styled.p`
+    font-size: 14px;
+    text-align: left;
+    color: white;
+`
+
+const DisplayPeople = props => {
+    console.log(props);
+    return (
```

Comment:
if you were to have passed the whole object from your People component, you can destructure even more to make your code concise here- you'd write something like like `const { name, gender, height, weight } = props.person` and then you could access them without having to prepend props each time, so for example just `Gender: {gender}`